packages/test-utils/src/testUtils/appWrappers.tsx

```diff
@@ -255,7 +255,8 @@ export const textContentMatcher =
       return false;
     }
 
-    const hasText = (textNode: Element) => textNode?.textContent === text;
+    const hasText = (textNode: Element) =>
+      textNode?.textContent?.includes(text) ?? false;
```

Comment:
I'm a little anxious about changing this -- it might be pretty heavily reused
```suggestion
    const hasText = (textNode: Element) => textNode?.textContent === text;
```

Reply:
OK then no worries!